src/data/systemIntake.ts

```diff
@@ -124,7 +128,10 @@ export const prepareSystemIntakeForApi = (systemIntake: SystemIntakeForm) => {
     existingContract: systemIntake.contract.hasContract,
     grtReviewEmailBody: systemIntake.grtReviewEmailBody,
     contractor: systemIntake.contract.contractor,
-    contractNumber: systemIntake.contract.number,
+    contractNumber:
+      systemIntake.contract.numbers.length > 0
+        ? systemIntake.contract.numbers.split(',').map(c => c.trim())
+        : [], // TODO(Sam): change `contractNumber` -> `contractNumbers`?
```

Comment:
@samoddball I'd sync with @mynar7 to see if this is still going to even be needed going forward. I would assume https://github.com/CMSgov/easi-app/pull/2459 would have deleted any endpoints that `PUT` or `POST` an entire intake. Perhaps we could follow up in another PR and delete the sagas on the frontend that we no longer need?

Reply:
ah-ha! That makes perfect sense - thank you for catching this!! Will chat with @mynar7 

Thanks!

Reply:
@ClayBenson94 @samoddball I tested the UI for that PR and noted the exact calls that are still using REST in the description. I would venture that this FE code is probably unnecessary and can be deleted.